modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZonesUtil.java

```diff
@@ -317,4 +321,29 @@ static long extractChangeTriggerRevision(Entry revisionEntry) {
             return INITIAL_TRIGGER_REVISION_VALUE;
         }
     }
+
+    /**
+     * Finds a zone configuration from zones configuration by its id.
+     *
+     * @param dstZnsCfg Distribution zones config.
+     * @param zoneId Id of zone.
+     * @return Zone configuration with appropriate zone id.
+     */
+    public static DistributionZoneConfiguration getZoneById(DistributionZonesConfiguration dstZnsCfg, int zoneId) {
+        if (zoneId == DEFAULT_ZONE_ID) {
+            return dstZnsCfg.defaultDistributionZone();
+        }
+
+        for (String name : dstZnsCfg.distributionZones().value().namedListKeys()) {
```

Comment:
Seems that we may use
```
ConfigurationUtil#getByInternalId(NamedConfigurationTree<T,?,?>, UUID)
ConfigurationUtil#getByInternalId(NamedListView<N>, UUID)
```

Reply:
Sorry, maybe sounds stupid, but I can't find these methods)